music21/spanner.py

```diff
@@ -752,10 +803,17 @@ def getLast(self):
 
 
 # ------------------------------------------------------------------------------
-class _SpannerRef(t.TypedDict):
+class PendingAssignmentRef(t.TypedDict):
+    '''
+    An object containing information about a pending first spanned element
+    assignment. See setPendingFirstSpannedElementAssignment for documentation
+    and tests.
+    '''
     # noinspection PyTypedDict
     spanner: 'Spanner'
```

Comment:
Remnant from before `from __future__ import annotations` can be changed to `Spanner` without quotes

Reply:
ok.